packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -26,6 +31,18 @@ import { DEFAULT_INSTRUMENTATION_PLUGINS, NodeTracerConfig } from './config';
 import { PluginLoader, Plugins } from './instrumentation/PluginLoader';
 import * as semver from 'semver';
 
+// TODO: uncomment when new version of @opentelemetry/propagator-jaeger is released
+// import { JaegerHttpTracePropagator } from '@opentelemetry/propagator-jaeger';
+
+api.propagation.registerNamedPropagator('b3', () => new B3SinglePropagator());
+api.propagation.registerNamedPropagator(
+  'b3multi',
+  () => new B3MultiPropagator()
+);
+
+// TODO: uncomment when new version of @opentelemetry/propagator-jaeger is released
```

Comment:
I think core should never depend on contrib. Can you update the comment to reflect that the jaeger propagator should be moved back into the core repo?

Reply:
Didn't we say during last SIG meetings, that it was fine to rely on Jaeger propagator from contrib?

Reply:
We said it was fine for jaeger propagators to be a dependency, but it should be moved into the main repo.

Reply:
Okay, will do.